airflow/contrib/hooks/gcp_bigtable_hook.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from google.cloud.bigtable import Client
+from google.cloud.bigtable.cluster import Cluster
+from google.cloud.bigtable.instance import Instance
+from google.cloud.bigtable.table import Table
+from google.cloud.bigtable_admin_v2 import enums
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+
+# noinspection PyAbstractClass
```

Comment:
Can we remove this

Reply:
We keep those comments to be able to have "warning free" code in IntelliJ. I find it pretty useful, so maybe we can leave it for now. 

Side comment - I think this really indicates a problem in underlying hook definition. Maybe we can fix it separately? Those not-implemented abstract methods are:  get_conn() get_records(), get_pandas_df() run(sql) - which for me call to have a separate abstract DBApiAbstractHook (it looks like those are all db api methds that are missing) rather than have them in the BaseHook. What do you think? I could open a separate JIRA issue for that one ?

Reply:
I agree, it is annoying to me as well. Can you please create a separate JIRA for it, we definitely need to solve this.

Reply:
Created: https://issues.apache.org/jira/browse/AIRFLOW-3629 . I will fix it shortly.

Reply:
For now I remove the abstract class knowing that I will fix it there.